lib/arena.rb

```diff
@@ -1,2 +1,79 @@
+require "pry"
+require_relative "./gladiator"
+
 class Arena
+  attr_accessor :name, :gladiators
+
+  def initialize name
+    @name = name.capitalize
+    @gladiators = []
+  end
+
+  def add_gladiator (gladiator)
+    unless gladiators.length == 2
+    gladiators << gladiator
+    end
+  end
+
+  def fight
+    if gladiators.length >= 2
+      if gladiators[0].name == "Maximus"
+        @gladiators.pop
+      elsif gladiators[1].name == "Maximus"
+        @gladiators.shift
+      elsif gladiators[0].weapon == "Spear" && gladiators[1].weapon == "Club"
+        the_vote
+      elsif gladiators[0].weapon == "Club" && gladiators[1].weapon == "Trident"
+        the_vote
+      elsif gladiators[0].weapon == "Trident" && gladiators[1].weapon == "Spear"
+        the_vote
+      elsif gladiators[0].weapon == gladiators[1].weapon
+        @gladiators = []
+      else
+        puts "thumbs up or thumbs down?"
+        up_or_down = gets.chomp
+        if up_or_down == "thumbs down"
+          @gladiators.shift
+        elsif up_or_down == "thumbs up"
+          @gladiators.pop
+        end
+      end
+    end
+  end
+
+  def remove_gladiator (name)
+    @gladiator.delete(name)
+  end
+
+  def entertained?
+    if @gladiators.include?("Maximus")
```

Comment:
Not sure if this works, think you could also use an enumerable `any?` to see if a gladiators name is Maximus